docs/report/report.tex

```diff
@@ -385,14 +385,60 @@ \subsubsection{Retrospektive}
 \colorbox{red}{TODO:}Was haben wir dabei gelernt? Neue Erkentnisse? Neue Sichtweisen?
 Was lief gut, neu gelernt, was lief nicht so gut, was verbessern?
 
-\section{Erkenntnisse}
-
 \section{Code Walkthrough}
 \colorbox{red}{TODO:}vielleicht interessante Code Snippets?
 
 \section{Testing}
 \colorbox{red}{TODO:}wie haben wir getestet, haben wir getestet?
 
+\section{Fazit}
+\subsection{Probleme, Lösungen, Erkenntnisse}
+\colorbox{red}{TODO: Finde TITEL:} Fazit oder nur Erkenntnisse oder oder?
+\colorbox{red}{Besseres Intro zum Kapitel?}
+Das Projekt Informatik gab einen Einblick in die Arbeitswelt, das Team musste sich klassischen Alltagsproblemen stellen, konnte Lösungswege finden und wichtige Erkenntnisse für die Zukunft daraus ziehen.
```

Comment:
```suggestion
Das Projekt Informatik gab einen realistischen Einblick in die Arbeitswelt. Das Team musste sich klassischen Alltagsproblemen stellen, Lösungswege finden und wichtige Erkenntnisse für die Zukunft daraus ziehen.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/419/commits/c5dae29b261f99d39aac66c889e6bb96205a2f1a